INTERPRETING.md

```diff
@@ -352,3 +352,30 @@ following strings:
 - **`generated`** The test file was created procedurally using the project's
   test generation tool. This flag is specified for informational purposes only
   and has no bearing on how the test should be interpreted.
+
+### `locale`
+
+The `locale` attribute allows tests to declare explicit information regarding locale specificity.
+
+- **`tags`** A list of valid language tags or subtags found in this test. Use IANA Language Subtag Registry to validate.
+
+  *Example*
+
+  ```js
+  /*---
+  locale: 
+    tags: [en, en-US, ar]
```

Comment:
That's what I was hoping we'd learn from @littledan.

Reply:
If we're being pedantic about things, there's all sorts of details that could be included here--which Intl constructors are actually supported on the locale, what all the locale data is, etc are not directly implied by the locale tag itself. I like the idea in this patch since it's *not* pedantically self-contained.

I like Leo's idea of using something like `locale: [en]` for this purpose. I can imagine lots of hypothetical extensions, but I don't have a particular use case for any in mind; this is already more elaborate than what I was initially picturing.

Whether or not we make that syntax change, let's make sure the INTERPRETING.md explains the sloppiness here, for example, adding the disclaimer:

> Tests with `locale` specified make assumptions about the specific locale data supported which go beyond what is implied by any specification. The information included is determined pragmatically based on common data for locales, e.g., from the CLDR. Implementations which use significantly different data may conform to the specification precisely but fail these tests.

Reply:
I'll be honest, your answer seems to say "I like both". 

I am trying to address this: 

> I was wondering, what if we made a feature flag behind which we could share Intl tests which assume a little bit about locale data? For example, it could be a feature flag for en-US data, and we could use our subjective judgment to keep the tests generally within what is unlikely to change over time


I don't think this information should be stored as a "feature" because it's not a new "feature" of the language in the same sense that e.g. SharedArrayBuffer is a new feature. 

I will change it to @leobalter's suggestion and then you can fill in the frontmatter of the relevant tests